### PR TITLE
Feature/2196: Provision new test users with roles 

### PR DESCRIPTION
**Pull request by cathysnider**
The new users are:
USERNAME - ROLE
osr-test-site-editor - site_editor
osr-test-edit-only - edit_only
osr-test-access-mgr - access_manager
osr-test-config-mgr - configuration_manager
(Form manager not added at this time)

QA: 
 - [x] New test users exist; are provisioned with role when they sign on
 - [x] Current test users continue to work

Test site: https://www-dev-new.colorado.edu/p1df671631d0/

### CODE REVIEW

File: tests/behat/features/Express/cu_users/invite.feature [hunk 1]
```diff
@@ -1,9 +1,97 @@
-@users @invite
+@users @javascript
```
Comment (alexfinnarn) [inline tests/behat/features/Express/cu_users/invite.feature]: Adding the `@javascript` tag here makes all the tests in this feature file run with the JS driver, which is really slow. I'm not sure any of these tests need the JS tag.

File: tests/behat/features/cu_people_bundle/1taxonomyandfilters.feature [hunk 2]
```diff
@@ -1,36 +0,0 @@
-# SEE ARTICLES FOR TAXONOMY PERMISSION TESTS
```
Comment (alexfinnarn) [inline tests/behat/features/cu_people_bundle/1taxonomyandfilters.feature]: These tests need to stay in until the bundle is removed.

File: tests/behat/features/cu_photo_gallery_bundle/photo_gallery/photo_gallery.access.feature [hunk 3]
```diff
@@ -1,40 +0,0 @@
-Feature: Photo Gallery Access
```
Comment (alexfinnarn) [inline tests/behat/features/cu_photo_gallery_bundle/photo_gallery/photo_gallery.access.feature]: bundle is still in the dev branch.

File: tests/behat/features/cu_social_media_bundle/context/context.feature [hunk 4]
```diff
@@ -1,11 +0,0 @@
-Feature: Context for the Social Media Bundle
```
Comment (alexfinnarn) [inline tests/behat/features/cu_social_media_bundle/context/context.feature]: Social Media bundle still in dev.

**Comment by jeor0980** [comment]
This PR no longer needed since we are not creating users on install.

**Pull request closed without merging**